vars/main.yml

```diff
@@ -39,17 +39,24 @@ nb_rest_port: 8080
 # Valid options:
 #   rpm_repo: Install ODL using its Yum repo config
 #   rpm_path: Install ODL from a local path or remote URL
-install_method: "rpm_repo"
+#   dep_repo: Install ODL using a debian repository
+install_method: "deb_repo"
 
 # URL of the .repo config to use when installing ODL from a repo
 # NB: This will only take effect when `install_method` is "rpm_repo"
 rpm_repo_file: "opendaylight-5-release.repo"
 rpm_repo_url: "https://git.opendaylight.org/gerrit/gitweb?p=integration/packaging.git;a=blob_plain;f=rpm/example_repo_configs/opendaylight-5-release.repo;hb=refs/heads/master"
 
+# URL of the deb repository to use when installing ODL from a repo
+# NB: This will only take effect when `install_method` is "deb_repo"
+deb_repo_url: "deb http://download.opensuse.org/repositories/home:/akshitajha/{{ os }}/ /"
+# URL to add the repository key to 'apt'
+deb_key: "http://download.opensuse.org/repositories/home:akshitajha/{{ os }}/Release.key"
+
 # This will be passed as the `name` param to the Ansible `yum` module.
 # `name` param docs: "You can also pass a url or a local path to a rpm file."
 #   See: http://docs.ansible.com/ansible/yum_module.html
 # NB: Local paths must be relative the host being configured (think `/vagrant`)
 # NB: This will only take effect when `install_method` is "rpm_path"
 # Default to the Boron RPM hosted on the CentOS Community Build System
-rpm_path: "http://cbs.centos.org/repos/nfv7-opendaylight-5-release/x86_64/os/Packages/opendaylight-4.2.0-1.el7.noarch.rpm"
+rpm_path: "http://cbs.centos.org/repos/nfv7-opendaylight-5-release/x86_64/os/Packages/opendaylight-4.2.0-1.el7.noarch.rpm"
```

Comment:
Nothing to do with your patch, but just saw that this line has a bug. Using 5-release repo to install ODL 4.2, which should fail.